setup.py

```diff
@@ -72,7 +72,6 @@ def setup_package():
             "Topic :: Scientific/Engineering :: GIS",
             "License :: OSI Approved :: BSD License",
             "Programming Language :: Python",
-            "Programming Language :: Python :: 3.6",
```

Comment:
Since we are only dropping 3.6+Windows testing should we retain the `"Programming Language :: Python :: 3.6"` label here?